scss/_variables.scss

```diff
@@ -95,12 +95,17 @@ $theme-colors: map-merge(
 $theme-color-interval:      8% !default;
 
 // The yiq lightness value that determines when the lightness of color changes from "dark" to "light". Acceptable values are between 0 and 255.
-$yiq-contrasted-threshold:  150 !default;
+$yiq-contrasted-threshold:  3 !default;
```

Comment:
Woups, should be 4.5!

Reply:
Fixed in [this commit](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/ed6aba7f78199906784bacaf4b870e1d3bfc8121).